edx_rest_framework_extensions/auth/jwt/authentication.py

```diff
@@ -78,6 +82,8 @@ def authenticate(self, request):
 
         except jwt.InvalidTokenError:
             raise exceptions.AuthenticationFailed()
+        except exceptions.NotAuthenticated:
```

Comment:
This feels like the right direction for allowing other auth to have a try.

Reply:
I just have concern that if it added a loop.

Reply:
Not sure about loops, but it seems like this isn’t needed for your current concern.